src/pages/run-a-node.js

```diff
@@ -0,0 +1,1011 @@
+            </h3>
+            <Collapse>
+              <ul>
+                <li>
+                  <Translation id="page-run-a-node-who-copy-1" />
+                </li>
+                <li>
+                  <Translation id="page-run-a-node-who-copy-2" />
+                </li>
+                <li>
+                  <Translation id="page-run-a-node-who-copy-3" />
+                </li>
+              </ul>
+              <h3>
+                <Translation id="page-run-a-node-who-copy-bold" />
+              </h3>
+            </Collapse>
+          </Width80>
+          <Width20>
+            <GatsbyImage image={getImage(data.impact)} />
+          </Width20>
+        </WhyHighlight>
+      </Content>
+
+      <Content id="why-run-a-node">
+        <h2>
+          <Translation id="page-run-a-node-why-title" />
+        </h2>
+        <InfoGrid>
+          {whyRunANodeCards.map(({ image, title, preview, body }, idx) => (
+            <ExpandableCard
+              contentPreview={<Translation id={preview} />}
+              title={<Translation id={title} />}
+              svg={image}
+              key={idx}
+            >
+              {body.map((item, idx) => (
+                <p key={idx}>{<Translation id={item} />}</p>
+              ))}
+            </ExpandableCard>
+          ))}
+        </InfoGrid>
+      </Content>
+
+      <Divider />
+
+      <Content id="getting-started">
+        <h2>
+          <Translation id="page-run-a-node-getting-started-hardware-title" />
+        </h2>
+        <p>
+          <Translation id="page-run-a-node-getting-started-hardware-1" />
+        </p>
+        <p>
+          <Translation id="page-run-a-node-getting-started-hardware-2" />
+        </p>
+        <MarginFlex>
+          <Container>
+            <h3>
+              <StyledEmoji text=":building_construction:" size={2} />
+              <Translation id="page-run-a-node-build-your-own-title" />
+            </h3>
+            <BuildFlex>
+              <BuildBox>
+                <h4>
+                  <Translation id="page-run-a-node-build-your-own-minimum-specs" />
+                </h4>
+                <ul>
+                  <li>
+                    <Translation id="page-run-a-node-build-your-own-min-ram" />
+                    <p>
+                      <ScrollLink onClick={() => scrollToId("plan-on-staking")}>
+                        <Translation id="page-run-a-node-build-your-own-ram-note-1" />
+                      </ScrollLink>
+                    </p>
+                    <p>
+                      <ScrollLink onClick={() => scrollToId("rasp-pi")}>
+                        <Translation id="page-run-a-node-build-your-own-ram-note-2" />
+                      </ScrollLink>
+                    </p>
+                  </li>
+                  <li>
+                    <Translation id="page-run-a-node-build-your-own-min-ssd" />
+                    <p>
+                      <small>
```

Comment:
This feels too much text to be a 'small',

Reply:
> Though you can still technically get away with 1 TB, the blockchain continues to grow over time and this will soon no longer be enough.

Also, with the statelessness and state-expiry roadmap, I'm not sure if we should mention this. The idea of having to continually upgrade your node hardware indefinitely might put some potential node runners off.

Reply:
Remove it and just leave "2 TB" as the recommended minimum?

I'd not against that, but if you're suggesting changing it to just 1 TB I'd be hesitant until statelessness/expiry are actually live. 

This is a tricky section, since each client is a little different, and future-proofing / trying to minimize future maintenance is somewhat subjective. In the end I'd prefer to keep it simple, so I'm down to just keep it at "2 TB SSD" and leave the note that SSD speeds are essential.

Reply:
I think we should say 2 TB and hope state bloat is fixed before they'll need to upgrade 😅

Reply:
You can also refer table under this https://ethereum.org/en/developers/docs/nodes-and-clients/#recommended-specifications
I would keep just 1TB as minimum, with prunning of old states, dropping ancient data, 1TB will be more than enough.

Reply:
I think I'm in favor of just recommending 2TB here. At the risk of excluding some people who are willing/able to get a 1 TB but not a 2 TB drive, recommending 2 will minimize future issues and complications in the attempt to keep this geared as much as possible towards non-technical users. This will also keep things really simple as we approach the merge, where the Beacon Chain will also need to be synced. 

In the future as statelessness, and history/state expiry become a reality, we can certainly readdress this minimum, and hopefully drop it entirely someday. In the meantime, of course more complex setups exist to bring cost down, but again this guide is intended to keep things as simple as possible, and I think this is a reasonable way to help accomplish this.

Reply:
Agree with @wackerow's last comment. I'd rather err on the side of caution.